Prace_domowe/Praca_domowa4/Grupa4/Marcin Wilk/PD4.ipynb

```diff
@@ -0,0 +1,1294 @@
+{
```

Comment:
<p>To nie jest dobry sposób na prezentację danych, w przeciwieństwie do boxplota powyżej, bo pierwsze CV ze skalowaniem ma się nijak do pierwszego CV bez skalowania</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/227/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>